contracts/contracts/interfaces/IRateLimiter.sol

```diff
@@ -53,13 +53,19 @@ interface IRateLimiter {
   error LimitIsZero();
 
   /**
-   * @notice Resets the rate limit amount to the amount specified.
-   * @param _amount sets the new limit amount.
+   * @notice Resets the rate limit amount.
+   * @dev If the used amount is higher, it is set to the limit to avoid confusion/issues.
+   * @dev Only the RATE_LIMIT_SETTER_ROLE is allowed to execute this function.
+   * @dev Emits the LimitAmountChanged event.
+   * @dev usedLimitAmountToSet will use the default value of zero if period has expired
```

Comment:
`usedLimitAmountToSet` is missing a dot at the end.

Reply:
done